packages/vitest/rollup.config.js

```diff
@@ -57,6 +57,8 @@ const external = [
   'node:worker_threads',
   'node:fs',
   'rollup',
```

Comment:
I think we should remove both of these from external - they should never be bundled. Maybe there is some kind of a mechanism to throw an error if import from rollup is detected? 🤔

Reply:
We just need to remove the rollup from `external`. If the rollup is bundled, the build and test will fail.